package.json

```diff
@@ -13,27 +13,56 @@
   ],
   "repository": "jshttp/cookie",
   "devDependencies": {
-    "beautify-benchmark": "0.2.4",
-    "benchmark": "2.1.4",
-    "eslint": "4.19.1",
-    "eslint-plugin-markdown": "1.0.0-beta.6",
-    "istanbul": "0.4.5",
-    "mocha": "1.21.5"
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/register": "^7.0.0",
+    "@pika/pack": "^0.3.2",
+    "@pika/plugin-build-node": "^0.3.11",
+    "@pika/plugin-build-web": "^0.3.11",
+    "@pika/plugin-standard-pkg": "^0.3.11",
+    "babel-plugin-istanbul": "^5.1.0",
+    "beautify-benchmark": "^0.2.4",
+    "benchmark": "^2.1.4",
+    "cross-env": "^5.2.0",
+    "eslint": "^5.13.0",
+    "eslint-plugin-markdown": "^1.0.0",
+    "mocha": "^5.2.0",
+    "nyc": "^13.3.0"
   },
   "files": [
     "HISTORY.md",
     "LICENSE",
     "README.md",
-    "index.js"
+    "src/index.js"
   ],
   "engines": {
     "node": ">= 0.6"
   },
   "scripts": {
     "bench": "node benchmark/index.js",
     "lint": "eslint --plugin markdown --ext js,md .",
-    "test": "mocha --reporter spec --bail --check-leaks test/",
-    "test-ci": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -- --reporter spec --check-leaks test/",
-    "test-cov": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter dot --check-leaks test/"
+    "test": "mocha --recursive --require @babel/register --bail --check-leaks",
+    "test-ci": "cross-env NODE_ENV=test nyc npm test",
```

Comment:
Can you please restore this back to `Istanbul`? Or at least can you explain why we need to change? Currently as far as I know `Istanbul` is working just fine, and it works on more versions of Node.js than `nyc` does, so helps us check coverage on our supported runtimes.

Reply:
Changed, turned back istanbul.